### ISSUE TITLE
[Feature request] Battery Level state or attribute

### ISSUE THREAD

**Issue by bridgewatermedia** [nitobuendia/oura-custom-component#18]
I'm not sure if the API has this exposed or not, but the battery level/charge level of the Oura ring would be a useful feature to have access to.  Is it possible to implement this in a future release?

Thanks!

**Comment by nitobuendia** [comment]
@bridgewatermedia Thanks a lot for opening the new issue.

My understanding is that Oura ring only syncs data to the cloud when you sync the ring on the app. This is also the data available on the API. As such, if you expected updated data on the battery status, this may not be possible at all since I assume you won't be syncing the app all the time.

Nonetheless, I've checked the [API documentation](https://cloud.ouraring.com/v2/docs#operation/sleep_route_sleep_get) and I only saw one reference to battery which is `"low_battery_alert"`. This exists within the [sleep periods endpoint](https://cloud.ouraring.com/v2/docs#operation/sleep_route_sleep_get) which is already available on the [Sleep Sensor](https://github.com/nitobuendia/oura-custom-component/tree/oura-v2#sleep-sensor) on oura-v2 that we're working on.

If your use case is simply to get a notification when battery is low, this may work - with the caveat that it might be updated only once in a day.
